samples/SampleApp/Program.cs

```diff
@@ -63,7 +63,7 @@ public void Main(string[] args)
                 _logger.LogWarning("Unexpected warning", ex);
             }
 
-            using (_logger.BeginScope("Main"))
+            using (_logger.BeginScopeImpl("Main"))
```

Comment:
Will change this sample (and others...like in Entropy) to use `BeginScope` logger extension once PR https://github.com/aspnet/Logging/pull/159 is checked in.